src/c_api/c_api_ndarray.cc

```diff
@@ -428,3 +436,41 @@ int MXCachedOpRegisterOpHook(NDArrayHandle handle,
   op->RegisterOpHook(clbk, monitor_all);
   API_END();
 }
+
+int MXNDArrayIsDeferredComputeEnabled(int *curr) {
```

Comment:
why dont we call this just `IsDeferredCompute`?

Reply:
`MX` prefix is required for symbol visibility. `NDArray` as this is part of the `NDArray` related functions.